test/test_collection.py

```diff
@@ -23,6 +23,8 @@
 from codecs import utf_8_decode
 from collections import defaultdict
 
+import bson
+
 sys.path[0:0] = [""]
 
 from bson import encode
```

Comment:
`encode` is already imported here so you can just use that on L809 instead of re-importing the entire `bson` module.